src/components/DashboardCard/DashboardCard.styled.ts

```diff
@@ -0,0 +1,13 @@
+import { Paper } from '@mui/material';
+import { styled } from '@mui/material';
+
+export const PaperStyled = styled(Paper)(() => {
```

Comment:
`DashboardCardStyled`

If we have more of these kinds of components, it will be difficult to know which one you want to import

Reply:
changed in the latest commit.